packages/admin-panel/src/widgets/InputField.js

```diff
@@ -100,6 +100,18 @@ export const InputField = ({
         />
       );
       break;
+    case 'textArray':
```

Comment:
Doesn't really matter that the array contents are text, does it? Maybe it does... But couple of options if it could work for, e.g., `[1, 2, 3]`
```suggestion
    case 'array':
```
```suggestion
    case 'jsonArray':
```

Reply:
Hmm, lemme do a quick experiment with other types and I'll update this if they work correctly

Reply:
Yep this works for other types, I've changed the named to `jsonArray`